README.md

```diff
@@ -1,9 +1,18 @@
-# S47_Effective_Killing
+4# S47_Effective_Killing
```

Comment:
The title of the README.md file seems to have a typo. It should start with a '#' for proper markdown formatting. [important]